data/pathfinder/paizo/roleplaying_game/ultimate_combat/uc_abilities_class.lst

```diff
@@ -1374,7 +1374,7 @@ Holy Gun			KEY:Paladin Archetype ~ Holy Gun			CATEGORY:Archetype	TYPE:Archetype.
+Knight of the Sepulcher	KEY:Antipaladin Archetype ~ Knight Of The Sepulcher	CATEGORY:Archetype	TYPE:Archetype.AntipaladinArchetype.PaladinCruelty.PaladinSmiteGoodUse10.PaladinAuraOfDespair.PaladinFiendishBoonAbility.PaladinAuraOfVengeance.PaladinAuraOfSin.PaladinUnholyChampion	PRECLASS:1,Antipaladin=1	PREMULT:1,[PREABILITY:1,CATEGORY=Archetype,Antipaladin Archetype ~ Knight of the Sepulcher],[!PREABILITY:1,CATEGORY=Archetype,TYPE.PaladinCruelty,TYPE.PaladinSmiteGoodUse10,TYPE.PaladinAuraOfDespair,TYPE.PaladinFiendishBoonAbility,TYPE.PaladinAuraOfVengeance,TYPE.PaladinAuraOfSin,TYPE.PaladinUnholyChampion]	DESC:Not content with the antipaladin's mere corruption of the soul, the knight of the sepulcher sacrifices mortality along with morality. The knight of the sepulcher archetype is available only to the antipaladin alternate class.	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Touch of the Crypt|PRECLASS:1,Antipaladin=5	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Fortitude of the Crypt|PRECLASS:1,Antipaladin=8	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Cloak of the Crypt|PRECLASS:1,Antipaladin=10	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Will of the Crypt|PRECLASS:1,Antipaladin=11	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Weapons of Sin|PRECLASS:1,Antipaladin=14	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Crypt Lord|PRECLASS:1,Antipaladin=15	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Soul of the Crypt|PRECLASS:1,Antipaladin=17	ABILITY:Antipaladin Class Feature|AUTOMATIC|Knight Of The Sepulcher ~ Undying Champion|PRECLASS:1,Antipaladin=20	COST:0	SOURCEPAGE:p.63	ASPECT:Archetype Base Class|Antipaladin
```

Comment:
Unless you have a conversion to change this for characters - this is a no-no change. It breaks existing characters.